### PR TITLE
test(suite_settings): mayfail tag removed for pass tests

### PR DESCRIPTION
**Pull request by StateOf-Vlado**
#9284

### Fix tests in the Settings suite for Mac

Test runs: 
https://ci.infra.status.im/job/status-desktop/job/nightly-e2e/20/
https://ci.infra.status.im/job/status-desktop/job/nightly-e2e/21/
https://ci.infra.status.im/job/status-desktop/job/nightly-e2e/22/

Please run on your machines, for my mac all tests pass, as for CI

### CODE REVIEW

**Comment by StateOf-Vlado** [comment]
Test results: https://ci.status.im/job/status-desktop/job/branches/job/linux/job/x86_64/job/tests-e2e/job/PR-10260/6/

**Review by anastasiyaig — APPROVED** [review]

**Review by saledjenic — APPROVED** [review]

**Review by noeliaSD — CHANGES_REQUESTED** [review]
The test `The user add a contact with a chat key` works locally for me but the ones from `tst_mainSettinsSection`  that switches state from offline/online don't. 

The behavior is a bit random, sometimes one passes sometimes fails.. There are no tests runtime errors neither lookup errors it is just that the verification fails bc the expected status is wrong. So the tests seem to be ok, but the app behavior no. It should be online/offline but it isn't. 

Here one example of failure:

Screenshot 2023-04-14 at 09 48 28

- Screenshot when it should be offline:
failed_1

- Screenshot when it should be online:
failed_2

I'm wondering at this point that maybe the app has a bugs on this functionality. I suggest to test some times manually to see if it is only reproducible with automation.

**Comment by StateOf-Vlado** [comment]
> The test `The user add a contact with a chat key` works locally for me but the ones from `tst_mainSettinsSection` that switches state from offline/online don't.
> 
> The behavior is a bit random, sometimes one passes sometimes fails.. There are no tests runtime errors neither lookup errors it is just that the verification fails bc the expected status is wrong. So the tests seem to be ok, but the app behavior no. It should be online/offline but it isn't.
> 
> Here one example of failure:
> 
> Screenshot 2023-04-14 at 09 48 28
> 
> * Screenshot when it should be offline:
>   failed_1
> * Screenshot when it should be online:
>   failed_2
> 
> I'm wondering at this point that maybe the app has a bugs on this functionality. I suggest to test some times manually to see if it is only reproducible with automation.

@noeliaSD , fixed. could you please test on your local machine?

**Comment by StateOf-Vlado** [comment]
test runs:
https://ci.infra.status.im/job/status-desktop/job/manual/11/
https://ci.infra.status.im/job/status-desktop/job/manual/9/
https://ci.infra.status.im/job/status-desktop/job/manual/8/

**Comment by noeliaSD** [comment]
> > The test `The user add a contact with a chat key` works locally for me but the ones from `tst_mainSettinsSection` that switches state from offline/online don't.
> > The behavior is a bit random, sometimes one passes sometimes fails.. There are no tests runtime errors neither lookup errors it is just that the verification fails bc the expected status is wrong. So the tests seem to be ok, but the app behavior no. It should be online/offline but it isn't.
> > Here one example of failure:
> > Screenshot 2023-04-14 at 09 48 28
> > 
> > * Screenshot when it should be offline:
> >   failed_1
> > * Screenshot when it should be online:
> >   failed_2
> > 
> > I'm wondering at this point that maybe the app has a bugs on this functionality. I suggest to test some times manually to see if it is only reproducible with automation.
> 
> @noeliaSD , fixed. could you please test on your local machine?

I still have the same issues randomly happening locally and screenshots really say that the state is not the correct one.. Again, maybe it is a real issue in the app.. ?!

And regarding the test that changes the password, I also have some random failures but I think this case is due to timeouts, bc when restarting, or when changing the password, or when login again, it takes several time and I think the test sometimes doesn't wait enough time!

**Comment by StateOf-Vlado** [comment]
> > > The test `The user add a contact with a chat key` works locally for me but the ones from `tst_mainSettinsSection` that switches state from offline/online don't.
> > > The behavior is a bit random, sometimes one passes sometimes fails.. There are no tests runtime errors neither lookup errors it is just that the verification fails bc the expected status is wrong. So the tests seem to be ok, but the app behavior no. It should be online/offline but it isn't.
> > > Here one example of failure:
> > > Screenshot 2023-04-14 at 09 48 28
> > > 
> > > * Screenshot when it should be offline:
> > >   failed_1
> > > * Screenshot when it should be online:
> > >   failed_2
> > > 
> > > I'm wondering at this point that maybe the app has a bugs on this functionality. I suggest to test some times manually to see if it is only reproducible with automation.
> > 
> > 
> > @noeliaSD , fixed. could you please test on your local machine?
> 
> I still have the same issues randomly happening locally and screenshots really say that the state is not the correct one.. Again, maybe it is a real issue in the app.. ?!
> 
> And regarding the test that changes the password, I also have some random failures but I think this case is due to timeouts, bc when restarting, or when changing the password, or when login again, it takes several time and I think the test sometimes doesn't wait enough time!

It should work, it is not a bug, because it works on my machine, so I increased waitFor to 5 sec, please rerun the test

File: test/ui-test/src/screens/components/cahange_password_popup.py [hunk 1]
```diff
@@ -16,4 +16,4 @@ def change_password(self, old_pwd: str, new_pwd: str):
         self._new_password_text_field.text = new_pwd
         self._confirm_password_text_field.text = new_pwd
         self._submit_button.click()
-        self._quit_button.click()
\ No newline at end of file
+        self._quit_button.wait_until_appears(15000).click()
```
Comment (StateOf-Vlado) [inline test/ui-test/src/screens/components/cahange_password_popup.py:19]: Waiting for popup

**Review by glitchminer — CHANGES_REQUESTED** [review]
@StateOf-Vlado, I've sent you an export of the results. I had failures in the following:

**TestCase tst_mainSettingsSection**
- Scenario The user can backup seed phrase
- Scenario The user can switch state to online
- Scenario The user can change the password and login with new password 

**TestCase tst_userIdentity**
- ScenarioOutline The user sets display name, bio and social links 

**TestCase tst_contactsFlow** 
- Scenario The user can add a contact from the chat

**Review by noeliaSD — APPROVED** [review]
Last tries on my local machine passed!

**Comment by StateOf-Vlado** [comment]
> @StateOf-Vlado, I've sent you an export of the results. I had failures in the following:
> 
> **TestCase tst_mainSettingsSection**
> 
> * Scenario The user can backup seed phrase
> * Scenario The user can switch state to online
> * Scenario The user can change the password and login with new password
> 
> **TestCase tst_userIdentity**
> 
> * ScenarioOutline The user sets display name, bio and social links
> 
> **TestCase tst_contactsFlow**
> 
> * Scenario The user can add a contact from the chat

fixed

**Comment by StateOf-Vlado** [comment]
> @StateOf-Vlado, I've sent you an export of the results. I had failures in the following:
> 
> **TestCase tst_mainSettingsSection**
> 
> * Scenario The user can backup seed phrase
> * Scenario The user can switch state to online
> * Scenario The user can change the password and login with new password
> 
> **TestCase tst_userIdentity**
> 
> * ScenarioOutline The user sets display name, bio and social links
> 
> **TestCase tst_contactsFlow**
> 
> * Scenario The user can add a contact from the chat



> 

@glitchminer test the current solution, is it reproduced now?

**Review by glitchminer — APPROVED** [review]
@StateOf-Vlado , for a while I got a similar intermittent result like @noeliaSD, but running 1 at a time seems OK.

**Pull request merged**